docs/reference/pages/theory.md

```diff
@@ -94,7 +94,7 @@ You can apply custom behavior to this process by overriding `Page` class methods
 Page publishing can be scheduled through the _Set schedule_ feature in the _Status_ side panel of the _Edit_ page. This allows you to set up initial page publishing or a page update in advance.
 For pages to go live at the scheduled time, you should set up the [publish_scheduled](publish_scheduled) management command.
 
-The basic workflow is as follows:
+### Basic Workflow for Scheduled Publishing
```

Comment:
```suggestion
### Basic workflow for scheduled publishing
```

Titles should be sentence case.

Please can you update the others.

Aside from that, I think this looks like a reasonable change, thank you.

Reply:
Thanks for the review @lb- ! I've applied the requested changes.

(Side note: I wasn't sure whether to use sentence case - recommended by style guide - or to use title case, which is used in the surrounding headings. I ended up using the latter for consistency.)

Reply:
Good call, let's try to align new work to the style guide, this page might be older. 

Feel free to do a seperate PR with some clean up of this page, we don't want to update every single page across all the docs in one go though.

Reply:
@lb- I'm keeping the titles here with sentence case and opening new PR to change the rest of the titles in this page, as per suggestion.